lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -114,35 +148,33 @@ def get(self):
             # Ignore anything that doesn't use our API prefix, because it's
             # not in our API.
             if url.startswith(self.uri_prefix):
-                # If the URI is parameterized with a Flask "<type:name>"
-                # template string, we don't want to report it, so we remove
-                # it from the URI. We derive an API name by converting the
-                # "/" characters in the URI to "_", after removing a trailing
-                # "/" that would have been left by removing a template... we
-                # don't remove the trailing "/" from the URI, which serves as
-                # an indication that the parameter is needed.
-                #
-                # E.g, "/api/v1/controllers/list" yields:
-                #     "controllers_list": "/api/v1/controllers/list"
-                #
-                # while "/api/v1/users/<string:username>" yields:
-                #     "users": "/api/v1/users/"
-                #
-                # TODO: This won't work right with embedded template strings,
-                # which we're not currently using anywhere; but it'll require
-                # adjustment later if we add any. (E.g., something like
-                # "/api/v1/foo/<string:name>/detail/<string:param>")
```

Comment:
It seems premature to remove this warning: the proposed approach still doesn't work if there is an embedded parameter in the middle of the URI (as in the example at old-line 134).

Either that, or we need an approach which truncates the API at the first parameter, instead of just removing the parameters.